media-libs/libjxl/libjxl-9999.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit xdg cmake git-r3
+
+DESCRIPTION="JPEG XL image format reference implementation"
+HOMEPAGE="https://github.com/libjxl/libjxl"
+
+EGIT_REPO_URI="https://github.com/libjxl/libjxl.git"
```

Comment:
Could you also add a release and not a pure-9999? You see -9999 can break *any* time by upstream while release is at least guaranteed to work on a state when it was merged. You can keep maintaining the -9999 if you wish in parallel, with:
 https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Handling_multi_live.2Frelease_ebuilds

It seems upstream is alive with frequent releases too.

Reply:
I know the `if [[ ${PV} == 9999 ]]; then` trick.

Unfortunately, the current and previous libjxl releases are not very maintainer friendly.

Ebuild for standard release will be much complicated. There is need for heavy patching and various workarounds to satisfy gentoo policies.

I was thinking to add it later.